src/site/content/en/blog/responsiveness/index.md

```diff
@@ -0,0 +1,302 @@
+durations, we are considering two potential approaches:
+
+- **Maximum event duration:** the interaction latency is equal to the largest
+  single event duration from any event in the interaction group.
+- **Total event duration:** the interaction latency is the sum of all event
+  durations, ignoring any overlap.
+
+As an example, the diagram below shows a _key press_ interaction that consists
+of a `keydown` and a `keyup` event. In this example there is a duration overlap
+between these two events. To measure the latency of the key press interaction,
+we could use `max(keydown duration, keyup duration)` or `sum(keydown duration,
+keyup duration) - duration overlap`:
+
+{% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/td8jMQBTjyR34ZO1QBvp.svg", alt="A
+diagram showing interaction latency based on event durations", width="800",
+height="424" %}
+
+There are pros and cons of each approach, and we'd like to collect more data and
+[feedback](#feedback) before finalizing a latency definition.
+
+{% Aside %}
+  The event duration is meant to be the time from the event hardware timestamp
+  to the time when the next paint is performed after the event is handled. But
+  if the event doesn't cause any update, the duration will be the time from
+  event hardware timestamp to the time when we are sure it will not cause any
+  update.
+{% endAside %}
+
+{% Aside %}
+  For keyboard interactions, we usually measure the `keydown` and `keyup`. But
+  for IME, such as input methods for Chinese and Japanese, we measure the input
+  events between a `compositionstart` and a `compositionend`.
+{% endAside %}
+
+## Aggregate all interactions per page
+
+Once we're able to measure the end-to-end latency of all interactions, the next
+step is to define an aggregate score for a page visit, which may contain more
+than one interaction.
+
+After exploring a number of options, we've narrowed our choices down to the
+strategies outlined in the following section, each of which we're currently
+collecting real-user data on in Chrome. We plan to publish the results of our
+findings once we've had time to collect sufficient data, but we're also looking
+for direct [feedback](#feedback) from site owners as to which strategy would
+most accurately reflect the interaction patterns on their pages.
+
+### Aggregation strategies options
+
+To help explain each of the following strategies, consider an example page visit
+that consists of four interactions:
+
+<div class="w-table-wrapper">
+  <table>
+    <tr>
+      <th>Interaction</th>
+      <th style="text-align: right">Latency</th>
+    </tr>
+    <tr>
+      <td>Click</td>
+      <td style="text-align: right">120 ms</td>
+    </tr>
+    <tr>
+      <td>Click</td>
+      <td style="text-align: right">20 ms</td>
+    </tr>
+    <tr>
+      <td>Key press</td>
+      <td style="text-align: right">60 ms</td>
+    </tr>
+    <tr>
+      <td>Key press</td>
+      <td style="text-align: right">80 ms</td>
+    </tr>
+  </table>
+</div>
```

Comment:
Forcing the second column to align right pushed that content off screen on mobile viewports.

![Screenshot 2021-11-02 at 16 48 11](https://user-images.githubusercontent.com/17127567/139882369-ae861f83-2e2b-490c-80c7-90f1615a82d2.png)